parsec/core/logged_core.py

```diff
@@ -34,8 +34,7 @@ class LoggedCore:
     local_db = attr.ib()
     event_bus = attr.ib()
     encryption_manager = attr.ib()
-    mountpoint = attr.ib()
```

Comment:
[This broke the GUI](https://github.com/Scille/parsec-cloud/pull/222#issuecomment-467816616).